Snowflake.Data.Tests/UnitTests/ConnectionPoolManagerTest.cs

```diff
@@ -160,30 +161,16 @@ public void TestSetTimeoutForAllPools()
         }         
         
         [Test]
-        public void TestSetPoolingDisabledForAllPools()
+        public void TestSetPoolingDisabledForAllPoolsNotPossible()
```

Comment:
I've commented on possibility of this in the production code

Reply:
done